src/dk/aau/cs/ds306e18/tournament/model/format/SingleEliminationFormat.java

```diff
@@ -104,43 +107,19 @@ private void placeTeamsInBracket(ArrayList<Team> seedList, ArrayList<Team> byeLi
         }
     }
 
-    /** Reorders seedList to a fair list order for seeding
-     * @param seedList a list of seeded teams in ascending order */
-    private void fairSeeding(ArrayList<Team> seedList) {
-        //Variables used for seeding
-        int slice = 1;
-        int interactions = seedList.size() / 2;
-
-        //Order the teams in a seeded list
-        while (slice < interactions) {
-            ArrayList<Team> temp = new ArrayList<>(seedList);
-            seedList.clear();
-
-            while (temp.size() > 0) {
-                int lastIndex = temp.size();
-                seedList.addAll(temp.subList(0, slice));
-                seedList.addAll(temp.subList(lastIndex - slice, lastIndex));
-                temp.removeAll(temp.subList(lastIndex - slice, lastIndex));
-                temp.removeAll(temp.subList(0, slice));
-            }
-
-            slice *= 2;
-        }
-    }
-
     /** Add the needed amount of byes
      * @param numberOfTeams the amount of teams in the stage
      * @return byeList, an arrayList containing dummy teams */
     private ArrayList<Team> addByes(int numberOfTeams){
-        int numberOfByes = (int) Math.pow(2, rounds) - numberOfTeams;
+        int numberOfByes = pow2(rounds) - numberOfTeams;
         ArrayList<Team> byeList = new ArrayList<>();
         while (byeList.size() < numberOfByes) {
             byeList.add(new Team("bye" + byeList.size(), new ArrayList<>(), 999, ""));
         }
         return byeList;
     }
 
-    /** Gives the matches labels. */
+    /** Gives the matches identifiers. */
```

Comment:
> 
> 
> It would be nice if the final extra match would switch the team colours of the participants. More of a nice-to-have.
> ![image](https://user-images.githubusercontent.com/11318702/52050833-0f8dad80-2552-11e9-8cbb-30aca17a8d2d.png)

I thought I already did that. Must have mixed it up. Should be easy to change! But not super relevant if we make #49